auto_py_to_exe/web/js/i18n.js

```diff
@@ -2136,6 +2204,10 @@ const supportedLanguages = [
     name: 'Greek (Ελληνικά)',
     code: 'el',
   },
+  {
+    name: 'Hebrew (עברית)',
```

Comment:
You checked "I have added the language to supportedLanguages in alphabetical order" - maybe you forgot to commit the change? It is not in this PR.

Reply:
yes. added it, between Hindi and Greek

Reply:
My bad, I was meant to reference "I have added the language to the table in README.md in alphabetical order". The changes need to also be in README.md - [in this block](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/README.md?plain=1#L130).